inventory/hosts

```diff
@@ -55,6 +55,19 @@ www.openfoodnetwork.in
 [in:children]
 in-prod
 
+#------------------------------------------------------------------------------
+# Italy
+
+#[it-prod]
+#openfoodnetwork.pixelwise.it
```

Comment:
Do we need to add it if it's not yet ready?

Reply:
Removed :+1: